src/Servant/EDE.hs

```diff
@@ -93,16 +94,20 @@ import qualified Data.Vector         as V
 -- This would try to load @home.tpl@, printing any error or
 -- registering the compiled template in a global (but safe)
 -- compiled template store, if successfully compiled.
+type Filters = (Text,Term)
```

Comment:
Could you move this type above or below? Otherwise the comment for `loadTemplates` will be displayed as documentation for `Filters`. By the way, shouldn't it be `Filter`?